### PR TITLE
Update NacosServiceDiscovery.java Remove useless code

### PR DESCRIPTION
**Pull request by Colin678**
Remove useless code

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [x] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [x] GitHub Actions works fine on your own branch.
- [x] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by AlbumenJ** [comment]
please resolve confilcts

**Comment by Colin678** [comment]
resolve confilcts @AlbumenJ

**Comment by BurningCN** [comment]
pls remove unused import 
image

**Comment by 24kpure** [comment]
Why not merge 5 commit into one？

**Comment by Colin678** [comment]
The interval is too long，is not a pr？ QwQ@24kpure

**Comment by 24kpure** [comment]
> The interval is too long，is not a pr？ QwQ@24kpure

You'd better  have a look at checklist.
`Each commit in the pull request should have a meaningful subject line and body`

**Comment by Colin678** [comment]
I give up ，I will find a more meaningful submission@24kpure

File: dubbo-registry/dubbo-registry-nacos/src/main/java/org/apache/dubbo/registry/nacos/NacosServiceDiscovery.java [hunk 1]
```diff
@@ -16,32 +16,27 @@
...
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 import static org.apache.dubbo.common.function.ThrowableConsumer.execute;
-import static org.apache.dubbo.registry.nacos.util.NacosNamingServiceUtils.createNamingService;
-import static org.apache.dubbo.registry.nacos.util.NacosNamingServiceUtils.getGroup;
-import static org.apache.dubbo.registry.nacos.util.NacosNamingServiceUtils.toInstance;
+import static org.apache.dubbo.registry.nacos.util.NacosNamingServiceUtils.*;
```
Comment (AlbumenJ) [inline dubbo-registry/dubbo-registry-nacos/src/main/java/org/apache/dubbo/registry/nacos/NacosServiceDiscovery.java:39]: Please do not use comma inport in Dubbo project

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

**Pull request closed without merging**